Mathlib/NumberTheory/LSeries/Linearity.lean

```diff
@@ -132,3 +130,57 @@ lemma LSeriesSummable.smul_iff {f : ℕ → ℂ} {c s : ℂ} (hc : c ≠ 0) :
 @[simp]
 lemma LSeries_smul (f : ℕ → ℂ) (c s : ℂ) : LSeries (c • f) s = c * LSeries f s := by
   simp only [LSeries, term_smul_apply, tsum_mul_left]
+
+/-!
+### Sums
+-/
+
+section sum
+
+variable {ι : Type*} [DecidableEq ι] (f : ι → ℕ → ℂ) (S : Finset ι) (s : ℂ)
+
+lemma LSeries.term_sum_apply (n : ℕ) :
+    term (∑ i ∈ S, f i) s n  = ∑ i ∈ S, term (f i) s n := by
+  induction S using Finset.induction_on with
```

Comment:
wouldn't it be simpler to do cases on `n` here rather than induction on `S`?

Reply:
True; will do.